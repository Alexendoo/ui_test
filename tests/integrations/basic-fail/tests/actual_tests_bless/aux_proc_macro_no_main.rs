//@aux-build:the_proc_macro.rs:proc-macro
//! Test that our automatic --crate-type detection (changing this crate to `lib`)
//! does not change the aux build to `lib`

use the_proc_macro::thing;

thing!(cake);
