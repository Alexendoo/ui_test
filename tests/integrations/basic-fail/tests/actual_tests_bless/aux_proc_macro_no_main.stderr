error: expected one of `!` or `::`, found `<eof>`
 --> $DIR/aux_proc_macro_no_main.rs:7:8
  |
7 | thing!(cake);
  |        ^^^^ expected one of `!` or `::`

error: aborting due to previous error

